packages/extension-ui/src/Popup/ImportQr.tsx

```diff
@@ -50,7 +50,6 @@ export default function ImportQr (): React.ReactElement<Props> {
           {name && (
             <ButtonArea>
               <Button
-                label='Add the account with identified address'
```

Comment:
Was it not showing in the UI at all? It's a graphics-only button, right? Maybe we should show it in the `alt` text.